app/controllers/triage_controller.rb

```diff
@@ -97,4 +99,14 @@ def set_patient_session
   def triage_params
     params.require(:triage).permit(:status, :notes)
   end
+
+  def redirect_path
+    if session[:current_section] == "vaccinations"
+      vaccinations_session_path(@session)
+    elsif session[:current_section] == "consents"
```

Comment:
I can't see where `session[:current_section] = "consents"` is set?

Reply:
Ya, I meant to take this out. Will remove.

Reply:
I added setting the current_section in consents now.